codequality/checkstyle/checkstyle.xml

```diff
@@ -164,12 +164,6 @@
         <!-- Checks the placement of left curly braces on types, methods and other blocks. -->
         <module name="LeftCurly"/>
 
-        <!-- Checks for long lines. -->
```

Comment:
This change is required by breaking change in checkstyle which is upgraded in Gradle.